javascript/leap.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Created by justin maurer on 7/27/16.
+ */
+
+var Year = function(yearInput) {
+    this.testYear = yearInput;
+};
+
+Year.prototype.isLeap = function(){
+    var year = this.testYear;
+    if (divisibleBy4(year)) {
+        if (divisibleBy100(year)) {
+            if (divisibleBy400(year)) {
+                return true;
```

Comment:
The several levels nested conditionals might be hard to follow for a coworker. There's likely ways of reducing the nesting, like potentially returning the result of your logical expressions themselves rather than explicit `true`/`false`.